resource-definitions/template-driver/volumes/README.md

```diff
@@ -0,0 +1,3 @@
+This section contains Resource Definitions example for handling Kubernetes [Volumes](https://kubernetes.io/docs/concepts/storage/volumes) by using the [`template`](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/template/) Driver to configure your own `PersistentVolume` implementation. You can [see this other example](https://developer.humanitec.com/examples/resource-definitions/volume-pvc/volumes/) if you want to use the `volume-pvc` Driver.
+
```

Comment:
Shouldn't this also mention the `emotyDir` example too?

Reply:
Good point, I tried something, but I'm sure @TobiasBabin will make this even better.